custom-tests.yaml

```
@@ -1513,7 +1513,7 @@ api:
       <%api.RTCDataChannel:channel%>
       var instance = new RTCDataChannelEvent('', {channel: channel});
   RTCDTMFToneChangeEvent:
-    __base: "new RTCDTMFToneChangeEvent('', {tone: ''});"
+    __base: "var instance = new RTCDTMFToneChangeEvent('', {tone: ''});"
```

Quotes can be skipped in YAML a lot of the time:

```suggestion
    __base: var instance = new RTCDTMFToneChangeEvent('', {tone: ''});
```